dev/breeze/src/airflow_breeze/commands/testing_commands.py

```diff
@@ -553,7 +553,7 @@ def _verify_parallelism_parameters(
 @option_use_packages_from_dist
 @option_use_xdist
 @option_verbose
-@click.argument("extra_pytest_args", nargs=-1, type=click.UNPROCESSED)
+@click.argument("extra_pytest_args", nargs=-1, type=click.Path(path_type=str))
```

Comment:
Does making this a Path inhibit folks from using it for it's originally intended use, which was passing more pytest options/flags?

Reply:
No. It's ok. It does not have `exists=True` so it does not check if the path exists, and path_type= `str` avoids conversion of the string passed to `Path`, so things like `--system x` will both be passed verbatim - I tested it.